subsys/http/src/main/resources/default-http.conf

```diff
@@ -10,8 +10,9 @@
 #user=
 
 ## Proxy settings
-#proxy.host=http://myproxy.com
-#proxy.port=8001
+proxy.host=http://proxy.squi-001.prod.iad2.dc.redhat.com
```

Comment:
Avoid to declare this here, since we will use internal repo for this configuration.

Reply:
@sswguo could you clarify this more? You mean this proxy.host will be used for some other purpose?

Reply:
Ah, sorry, I just mean the value, avoid to commit the redhat one, should be good with `http://myproxy.com` , and let's put the redhat one in our ansible script.